geeksay.js

```diff
@@ -48,6 +48,7 @@ const translations = {
     "refresh": "f5",
     "slow": "O(n^n)",
     "fast": "O(1)",
+    "minute": "epoch",
```

Comment:
?

Reply:
I should add something to translations if I need it to get replaced right or did I get that wrong?

Reply:
If someone reads epoch in the translated sentence, would he understand the original word as minute?

Reply:
Not really! Let me remove that.